<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StegoSum</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QScrollArea" name="scrollArea">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>16</y>
             <width>198</width>
             <height>258</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="ClickableLabel" name="ImageView">
              <property name="text">
               <string> cover image</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>198</width>
             <height>291</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="ClickableLabel" name="ImageStegoView">
              <property name="text">
               <string>stego image</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QTabWidget" name="TabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="SteganographyTab">
         <attribute name="title">
          <string>Steganography</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QPushButton" name="encodeButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Encode</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="decodeButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Decode</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>password:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="openDataButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="acceptDrops">
                 <bool>false</bool>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>load secret data</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="default">
                 <bool>true</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="secretMsgTextEdit" name="secretMsgView">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="plainText">
                 <string>write secret message here ...</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QRadioButton" name="radioButtonText">
                  <property name="text">
                   <string>text</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButtonData">
                  <property name="text">
                   <string>data</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="copmressLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>compression:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="compressLabelChange">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>none</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QSlider" name="compressSlider">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="maximum">
                 <number>9</number>
                </property>
                <property name="pageStep">
                 <number>9</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
                <property name="tracking">
                 <bool>true</bool>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="invertedAppearance">
                 <bool>false</bool>
                </property>
                <property name="invertedControls">
                 <bool>false</bool>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::NoTicks</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="encryptCheckBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>encrypt</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="encodeMaxCheckBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <stylestrategy>PreferDefault</stylestrategy>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string> 3 channel encode</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QRadioButton" name="red_radio">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string comment="&lt;b&gt;red&lt;/b&gt;" extracomment="&lt;b&gt;red&lt;/b&gt;">red</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">buttonGroup</string>
                  </attribute>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="green_radio">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>green</string>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">buttonGroup</string>
                  </attribute>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="blue_radio">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>blue</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>false</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">buttonGroup</string>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QRadioButton" name="all_radio">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>all</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="SteganalysisTab">
         <attribute name="title">
          <string>Steganalysis</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,0,0">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Methods: </string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="samplePairsCheckBox">
              <property name="text">
               <string>Sample Pairs</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="makeAnalysisButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Make an analysis</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPlainTextEdit" name="consoleOutput"/>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_6" columnstretch="0,0,0,0,0">
            <item row="0" column="4">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="3">
             <widget class="QCheckBox" name="graphOutputCheckBox">
              <property name="text">
               <string>Graph</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QCheckBox" name="fileOutputcheckBox">
              <property name="text">
               <string>.txt File</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="consoleOutputCheckBox">
              <property name="text">
               <string>Console</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Outputs:  </string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Image">
    <property name="title">
     <string>&amp;Image</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_In_25"/>
    <addaction name="actionZoom_Out_25"/>
    <addaction name="action_Normal_Size"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Stego_Image"/>
   </widget>
   <addaction name="menu_Image"/>
   <addaction name="menu_View"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionEncode">
   <property name="text">
    <string>Encode</string>
   </property>
  </action>
  <action name="actionDecode">
   <property name="text">
    <string>Decode</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom &amp;In(25%)</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom &amp;Out(25%)</string>
   </property>
  </action>
  <action name="actionNormal_Size">
   <property name="text">
    <string>&amp;Normal Size</string>
   </property>
  </action>
  <action name="actionFit_to_Screen">
   <property name="text">
    <string>&amp;Fit to Window</string>
   </property>
  </action>
  <action name="actionZoom_In_25">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom &amp;In (25%)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out_25">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom &amp;Out (25%)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_Normal_Size">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Normal Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N, Ctrl+S</string>
   </property>
  </action>
  <action name="action_Fit_to_Window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Fit to Window</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Stego_Image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Stego Image</string>
   </property>
  </action>
  <action name="action_Stego_Image_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Stego Image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>secretMsgTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>secretmsgtextedit.h</header>
   <slots>
    <slot>textChangedHandler()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>decodeButton</sender>
   <signal>clicked()</signal>
   <receiver>actionDecode</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>encodeButton</sender>
   <signal>clicked()</signal>
   <receiver>actionEncode</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
